Userland/Libraries/LibWeb/HTML/HTMLTableCellElement.cpp

```diff
@@ -67,6 +79,19 @@ void HTMLTableCellElement::apply_presentational_hints(CSS::StyleProperties& styl
             return;
         }
     });
+    auto table_element = table_containing_cell(this);
```

Comment:
could we look up the table container only after confirming the border property is specified to prevent unnecessary work?